README.md

```diff
@@ -390,25 +385,20 @@ jobs:
     - uses: Azure/docker-login@v1
       with:
         login-server: contoso.azurecr.io
-        username: ${{ env.CR_USERNAME }}
-        password: ${{ env.CR_PASSWORD }}
+        username: ${{ secrets.REGISTRY_USERNAME }}
+        password: ${{ secrets.REGISTRY_PASSWORD }}
     
     - run: |
-        docker build . -t contoso.azurecr.io/k8sdemo:${{ github.sha }} --label dockerfile-path=./Dockerfile
+        docker build . -t contoso.azurecr.io/k8sdemo:${{ github.sha }} --label dockerfile-path=https://github.com/Username/RepoName/blob/branchOrTag/Dockerfile

```

Comment:
dockerfile-path=https://github.com/${{github.repo}}/blob/${GITHUB_REF##*/}/Dockerfile

Reply:
What does `${GITHUB_REF##*/}` indicate ?

Reply:
GITHUB_REF is an environment variable right. It extracts out the branch name like this:

![image](https://user-images.githubusercontent.com/25762922/102798000-7d595180-43d6-11eb-81f8-f13b56c4e760.png)

Reply:
Thanks for the suggestion, I will change that.